src/Grpc/Interop/test/InteropTests/InteropTests.csproj

```diff
@@ -15,5 +15,6 @@
       SkipGetTargetFrameworkProperties="true" />
 
     <TestAssetProjectReference Include="..\testassets\**\*.csproj" />
+    <TestAssetProjectReference Include="..\..\..\HttpApi\**\*.csproj" />
```

Comment:
This feels weird given the test project doesn't actually care about the new HttpApi files. Could you instead add a new project that just manages building the new set of projects❔

Reply:
Sure. I'm adding it here just to get something working.

If a new project is added then it will be empty and does nothing but have `<TestAssetProjectReference Include="..\..\..\HttpApi\**\*.csproj" />`.